events-processor/processors/processors.go

```diff
@@ -103,10 +103,11 @@ func StartProcessingEvents() {
 		With("service", "post_process")
 	slog.SetDefault(logger)
 
-	if os.Getenv("ENV") == "production" {
+	otelEndpoint := os.Getenv("OTEL_EXPORTER_OTLP_ENDPOINT")
+	if os.Getenv("ENV") == "production" && otelEndpoint != "" {
```

Comment:
We could even get rid of the env check:

```suggestion
	if otelEndpoint != "" {
```

?

Reply:
your call, it works for me